plugins/system/jacademyseo_opengraph/language/en-GB/plg_system_jacademyseo_opengraph.ini

```
@@ -0,0 +1,67 @@
+; JacademySEO Open Graph Plugin Language File
+; Copyright (C) 2025 JacademySEO. All rights reserved.
+; License GNU General Public License version 2 or later
+
+; Plugin Description
+PLG_SYSTEM_JACADEMYSEO_OPENGRAPH_DESCRIPTION="JacademySEO Open Graph Plugin automatically generates Open Graph tags for improved social media sharing. Features hierarchical override system supporting global, category, and article-level metadata configuration."
```

```suggestion
PLG_SYSTEM_OPENGRAPH_DESCRIPTION="Open Graph Plugin automatically generates Open Graph tags for improved social media sharing. Features hierarchical override system supporting global, category, and article-level metadata configuration."
```
As the plugin should go directly into the core, please adjust the language strings and remove JACADEMYSEO